iOS Client/Models/MBDevotion.swift

```diff
@@ -9,9 +9,64 @@
 import Foundation
 
 
-struct MBDevotion: Codable {
+struct MBDevotion: Codable, Detailable {
     var date: String
     var text: String
     var verse: String
     var verseText: String
 }
+
+struct LoadedDevotion: Codable, Detailable {
+    var date: String
+    var text: String
+    var verse: String
+    var verseText: String
+    var read: Bool
+    
+    init(devotion: MBDevotion, read: Bool) {
+        date = devotion.date
+        text = devotion.text
+        verse = devotion.verse
+        verseText = devotion.verseText
+        self.read = read
+    }
+    
+    var devotion: MBDevotion {
```

Comment:
where is this used? We should never really go back once we have our domain model.

Reply:
I don't think I need this I can remove